src/index.html

```diff
@@ -10,6 +10,18 @@
 	<link rel="icon" href="./assets/favicon.ico" />
 	<link rel="stylesheet" type="text/css" href="./assets/prism.css" />
 
+	<script>
```

Comment:
Need to manually include this for the time being - as the site is static and the service worker registration is included in the js bundle...